### DISCUSSION TITLE
支持 TransUnet 模型

### DISCUSSION THREAD

**Discussion by vansinhu** [open-mmlab/OpenMMLabCamp#155] [MMSegmentation]
# 支持 TransUnet 模型 

在 MMSegmentation projects 中添加  TransUnet 模型，需要和原始开源代码库[https://github.com/Beckschen/TransUNet](https://github.com/Beckschen/TransUNet) 中的模型推理精度一致
- 详细可见[https://github.com/open-mmlab/mmsegmentation/issues/2577](https://github.com/open-mmlab/mmsegmentation/issues/2577) 

## Task Information

**Repository**：MMSegmentation

**Difficulty**：Hard

**Points**：50

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：enemy1205

**Claim Time**：2023年07月16日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by enemy1205** [comment]
Claim the mission

**Reply by vansin** [reply]
wow~
